### PR TITLE
Release updates on accountchooser.com warnings & fix some issues

### PR DESCRIPTION
**Pull request by xil222**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+* Remove usage of goog.isFunction.
+* Fix FirebaseUI-web spinner off for RTL languages.
+* Show warning on accountchooser.com becomes universally opt-out.
+* Add unit test to confirm that if accountchooser.com JS dependency fails to load that the sign in flow would continue to work as expected.
```
Comment (bojeil-google) [inline CHANGELOG.md:4]: We can remove this one (unit test). We just need to focus on external facing changes.

File: CHANGELOG.md [hunk 2]
```diff
@@ -0,0 +1,4 @@
+* Remove usage of goog.isFunction.
```
Comment (bojeil-google) [inline CHANGELOG.md:1]: We can remove this one since it is not user facing change.

File: CHANGELOG.md [hunk 3]
```diff
@@ -0,0 +1,4 @@
...
+* Fix FirebaseUI-web spinner off for RTL languages.
```
Comment (bojeil-google) [inline CHANGELOG.md:2]: * Fixes FirebaseUI-web spinner for RTL languages.

File: CHANGELOG.md [hunk 4]
```diff
@@ -0,0 +1,4 @@
...
+* Show warning on accountchooser.com becomes universally opt-out.
```
Comment (bojeil-google) [inline CHANGELOG.md:3]: * Log warning on accountchooser.com being deprecated and switching to universal opt-out mode starting in July 31st, 2020.

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,4 +1,2 @@
-* Remove usage of goog.isFunction.
-* Fix FirebaseUI-web spinner off for RTL languages.
-* Show warning on accountchooser.com becomes universally opt-out.
-* Add unit test to confirm that if accountchooser.com JS dependency fails to load that the sign in flow would continue to work as expected.
\ No newline at end of file
+* Fixes FirebaseUI-web spinner off for RTL languages.
```
Comment (bojeil-google) [inline CHANGELOG.md:1]: You didn't fix this. You need to remove the word "off"
* Fixes FirebaseUI-web spinner for RTL languages.

**Comment by areegqween2** [comment]
@bojeil-google

**Pull request merged**